api-reference/openapi.json

```diff
@@ -1,23 +1,27 @@
 {
   "openapi": "3.1.0",
   "info": {
-    "title": "Wondercraft.ai API documentation",
-    "description": "You can find\nout more about Wondercraft.ai\n[here](https://www.wondercraft.ai/).\n",
-    "termsOfService": "https://support.wondercraft.ai/hc/en-gb/articles/14294433754258-Terms-of-Service",
-    "contact": { "email": "team@wondercraft.ai" },
-    "version": "0.1.0"
+    "title": "Wondercraft Public API",
+    "version": "0.0.1"
+  },
```

Comment:
lol - we can also keep 0.1.0.. 
I basically replaced this whole file with the `openapi.json` produced from the set of endpoints we only care about exposing.